spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/ColumnRef.scala

```diff
@@ -95,3 +95,18 @@ case object RowCountRef extends ColumnRef {
   override def cqlValueName: String = "count"
   override def cql: String = "count(*)"
 }
+
+/** References a function call **/
+case class FunctionCallRef(columnName: String,
+                           actualParams: Seq[Either[ColumnRef, String]] = Seq.empty,
```

Comment:
Little syntax thing here, we do an endline before parameters on multiline headers 
```scala
case class FunctionCallRef(
  columnName: String,
  actualParams: Seq[Either[ColumnRef, String]] = Seq.empty,
```